JSON_for_IO-Link.yaml

```diff
@@ -1605,75 +1296,64 @@ paths:
           content:
             application/json:
               schema:
-                type: array
-                items:
-                  type: object
-                  required:
-                    - portNumber
-                    - statusInfo
-                    - deviceAlias
-                  properties:
-                    portNumber:
-                      type: number
-                    deviceAlias:
-                      type: string
-                    statusInfo:
-                      type: string
-                      enum:                    # see Terminology in V.113, Table E.4 PortStatusList
-                        - COMMUNICATION_LOST   # NO_DEVICE
```

Comment:
Why are the enum definitions deleted?

Reply:
@mrentsch65 Maybe I understand it wrong but it shall still be there see $ref: '#/components/schemas/mastersPortsGet'

Reply:
Ok!